src/components/decoration/hero.tsx

```diff
@@ -100,7 +101,13 @@ const Hero: React.FC<HeroProps> = ({
           'object-cover',
           classNameMapping?.backgroundImage,
           !classNameMapping?.backgroundImage?.includes('object-') &&
-            (main ? 'object-top' : objectPositionMap[imageAlignment]),
+            (main
```

Comment:
```
main ? 'object-top' : objectPositionMap[imageAlignment] ?? `object-[${imageAlignment}]`
```

Reply:
Gives an error `Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ top: string; right: string; left: string; center: string; }'.`

Reply:
You're right, please also update the type of the objectPositionMap:
```
  const objectPositionMap: Record<string, string> = {
    top: 'object-top',
    right: 'object-right',
    left: 'object-left',
    center: 'object-center',
  };
```